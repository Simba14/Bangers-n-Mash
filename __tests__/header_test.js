import React from 'react';
import Header from '../src/components/Header';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import TestUtils from 'react-dom/test-utils';

test('correctly renders the Header component', () => {
  const tree = renderer.create(
    <Header />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

function setup() {
  const enzymeWrapper = mount(<Header {...'Boiler Room Picks'}/>)

  return {
    enzymeWrapper
  }
}

test('renders a h1', () => {
  const { enzymeWrapper } = setup()

  expect(enzymeWrapper.find('h1').text()).toBe('');
});
