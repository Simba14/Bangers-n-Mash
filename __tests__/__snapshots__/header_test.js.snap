// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders the Header component 1`] = `<h1 />`;
